WalletWasabi/WabiSabi/Client/WabiSabiHttpApiClient.cs

```diff
@@ -67,15 +69,52 @@ public async Task<RoundState[]> GetStatusAsync(CancellationToken cancellationTok
 		public Task ReadyToSignAsync(ReadyToSignRequestRequest request, CancellationToken cancellationToken) =>
 			SendAndReceiveAsync<ReadyToSignRequestRequest>(RemoteAction.ReadyToSign, request, cancellationToken);
 
+		private async Task<HttpResponseMessage> SendWithRetriesAsync(RemoteAction action, string jsonString, CancellationToken cancellationToken, int maxAttempts = 3)
```

Comment:
`CancellationToken` has to be the last parameter in the list. I think it could be ok to hardcode the max attempt to 3 instead on having it as a parameter btw.